blocks/api/parser.js

```diff
@@ -88,6 +53,71 @@ export function asType( value, type ) {
 }
 
 /**
+ * Returns an hpq matcher given a source object
+ *
+ * @param  {Object}   sourceConfig Attribute Source object
+ * @return {Function}              hpq Matcher
+ */
+export function matcherFromSource( sourceConfig ) {
+	switch ( sourceConfig.source ) {
+		case 'attribute':
+			return attr( sourceConfig.selector, sourceConfig.attribute );
+		case 'property':
+			return prop( sourceConfig.selector, sourceConfig.property );
+		case 'html':
+			return html( sourceConfig.selector );
+		case 'text':
+			return text( sourceConfig.selector );
+		case 'children':
+			return children( sourceConfig.selector );
+		case 'node':
+			return node( sourceConfig.selector );
+		case 'query':
+			const subMatchers = keys( sourceConfig.query ).reduce( ( memo, key ) => {
```

Comment:
Minor: Seems another one that could be a `mapValues`, even simplified to the function reference as callback:

```js
const subMatchers = mapValues( sourceConfig.query, matcherFromSource );
```

Reply:
Yep, actually I just discovered this function and it's awesome

Reply:
>Yep, actually I just discovered this function and it's awesome

Did you intend to make this change here?